### PR TITLE
[CSL-3370] Added TrackedItemPurchased type

### PR DESCRIPTION
**Pull request by HFaifman**
This PR adds a new type with the missing properties for an item during the Purchase event.
It also fixes an import which was breaking the existing type tests.

### CODE REVIEW

File: src/types/index.d.ts [hunk 1]
```diff
@@ -338,3 +338,8 @@ export interface ItemTracked {
   itemId?: string;
   variationId?: string;
 }
+
+export interface ItemTrackedPurchased extends ItemTracked{
```
Comment (esezen) [inline src/types/index.d.ts:342]:
```suggestion
export interface ItemTrackedPurchased extends ItemTracked {
```
Can we add a space before `{`?

Comment (HFaifman) [reply src/types/index.d.ts]: Yes, sure! Not exactly sure why Prettier didn´t pick on that though, will need to check.

File: src/types/autocomplete.d.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ import {
   RequestFeatureVariant,
   UserParameters,
   VariationsMap,
+  FilterExpression,
```
Comment (mocca102) [inline src/types/autocomplete.d.ts:8]: Thanks for catching this

**Review by mocca102 — APPROVED** [review]
LGTM!

**Pull request merged**